src/utils.ts

```diff
@@ -245,3 +252,81 @@ export function validateCoverageMetrics(
 
     return failures;
 }
+
+function getFormattedMetricData(coverageData: StatsFieldDataType) {
+    const { skipped, covered, pct } = coverageData;
+    return {
+        missed: { ...DEFAULT_METRIC_VALUE.missed, value: skipped },
+        covered: { ...DEFAULT_METRIC_VALUE.covered, value: covered },
+        percentage: { ...DEFAULT_METRIC_VALUE.percentage, value: pct },
+    };
+}
+
+function getAggregatedMetrics(metricsList: any[]) {
+    const aggData: any = {
+        lines: { total: 0, covered: 0, skipped: 0 },
+        branches: { total: 0, covered: 0, skipped: 0 },
+        functions: { total: 0, covered: 0, skipped: 0 },
+    };
+
+    metricsList.forEach((metric: any) => {
```

Comment:
No need to type `metric` here. If `metricsList` is typed correctly, then this would be inferred.